.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.4
+      - image: circleci/python:3.5
```

Comment:
https://circleci.com/gh/alphadose/coala/27#tests/containers/0 and https://circleci.com/gh/alphadose/coala/27#tests/containers/1 both show

```
$ python --version
Python 3.4.8
```